web3.js/.eslintrc.js

```diff
@@ -31,14 +31,6 @@ module.exports = {
         'newlines-between': 'always',
       },
     ],
-    indent: [
-      'error',
-      2,
-      {
-        MemberExpression: 1,
-        SwitchCase: 1,
-      },
-    ],
```

Comment:
I removed this eslint rule because indentation is already handled by prettier, and the rule conflicts with prettier's formatting.